### DISCUSSION TITLE
Didn't receive an e-mail when joined in waitlist.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#12159] [Copilot Conversations]
Hello there. I'm waiting for access to Github Copilot service for around 2 months now, and yes, I joined the waitlist. 

I know that there are a lot of people requesting access and not everyone can have access to it, but what makes me worried is that I didn't receive any e-mail when joined the waitlist. At first I thought it was a problem with the e-mail provider, but now, I don't know.

Is it normal guys? Thanks for the attention. ❤️ 😃

**Comment by davecheney** [comment]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Reply by ghost** [reply]
> I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

Yes, I saw that. I was just wondering if that's okay to join the waitlist but do not get any email telling that you joined or something...

**Reply by davecheney** [reply]
You can confirm your status by visiting https://github.com/features/copilot/signup

**Comment by holasoymalva** [comment]
I have been waiting for some months, but finally Copilot is already enabled on my account! I recommend you wait, since it is a process that cannot be expedited, and review the weekends to see if the co-driver is already available in your account, 🙂  so good luck getting in. 🙌

https://github.com/features/copilot/signup

**Reply by ghost** [reply]
I'm reviewing it today (a weekend xD) and still in the waitlist. I'll be patient, thanks! Happy that you already got it 😄

**Comment by pchettri3** [comment]
same here. Would be nice to know in email if the submission has gone through or not
